src/themes/teams/components/Chat/chatItemStyles.ts

```diff
@@ -1,7 +1,32 @@
-import { ICSSInJSStyle } from '../../../types'
+import { ICSSInJSStyle, ComponentSlotStylesInput } from '../../../types'
+import { ChatItemProps } from 'src/components/Chat/ChatItem'
+import { ChatMessageVariables } from './chatMessageVariables'
+import { pxToRem } from 'src/lib'
 
-const chatItemStyles = {
-  root: (): ICSSInJSStyle => ({}),
+const _10pxAsRem_ = pxToRem(10)
+const _14pxAsRem_ = pxToRem(14)
+
+const chatItemStyles: ComponentSlotStylesInput<ChatItemProps, ChatMessageVariables> = {
+  root: ({ props: p }): ICSSInJSStyle => ({
+    ...(p.gutter && { display: 'flex' }),
+    position: 'relative',
+    justifyContent: p.mine ? 'flex-end' : 'flex-start',
+    marginTop: _14pxAsRem_,
```

Comment:
Is this something that is becoming convention for using? It's kind a different that the conventions we have for general naming of variables...

Reply:
I was suggested to name them like this in other PRs